test/controllers/gems_adoptions_controller_test.rb

```diff
@@ -31,10 +31,18 @@ def setup
     assert_redirected_to gem_path(id: 1)
   end
 
+  test "should close a gem up for adoption" do
+    adoption_requests = adoption_requests(:one)
```

Comment:
This should probably be singular, since the variable only represents one `adoption_request`.